doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,194 @@
+802.15.4 Radio Driver)
+
+When either CX variant is enabled, the driver requests access to the RF medium from the PTA and informs it about the operation it's about
+to perform. It also reacts properly to the information from the PTA that access to RF medium was granted/denied.
+
+Generic 3-pin interface
+=======================
+
+This implementation uses the following pins to communicate with the PTA:
+
+* REQUEST pin - output controlled by the protocol driver, it is asserted in order to request RF access from the PTA. It can be configured by
+  setting the `req-gpios` property of the `nrf_radio_coex` device tree node.
+* PRIORITY pin - output controlled by the protocol driver, multiplexes the information about the operation priority and the type of operation
+  (RX/TX) to perform. It can be configured by setting the `pri-dir-gpios` property of the `nrf_radio_coex` device tree node. Currently only
+  priority "low" is supported.
+* GRANT pin - output controlled by the PTA, asserts, when PTA grants access to RF to the 802.15.4 and deasserts when it denies the access. It can
+  be configured by setting the `grant-gpios` property of the `nrf_radio_coex` device tree node.
+
+The support for this interface is provided in a single-file plugin located in the NRF Connect SDK nrf repo, in the subsys/mpsl/cx directory.
+It is only available to use with the Zephyr RTOS.
+
+Adding support for CX generic 3-pin interface
+---------------------------------------------
+
+An out of the box supported type of PTA is the so called "generic 3-pin interface".
+
+To use the generic 3-pin interface PTA, complete the following steps:
+
+1. Set the :kconfig:`CONFIG_MPSL_CX_GENERIC_3PIN` Kconfig option to ``y``.
+
+2. Add the device tree `nrf_radio_coex` node in the way as described in :ref:`Hardware description`.
+
+After completing these steps no other function calls in the application are needed and the Coex will work.
+
+
+"Thread" implementaion
+======================
+
+This implementation is similar to the :ref:`Generic 3-pin interface`, however it is the REQUEST pin which contains information about the
+transfer direction and it is not multiplexed by using the PRIORITY pin (the device tree parameter `pri-dir-gpios` name is misleading here) This
+implementation uses the following pins to communicate with the PTA:
+
+* REQUEST pin - output controlled by the protocol driver, it is asserted when the SR driver intends to perform a TX operation and deaserted if the
+  operation is RX or the driver does not intend to perform any operation. It can be configured by setting the `req-gpios`
+  property of the `nrf_radio_coex` device tree node.
+* PRIORITY pin - output controlled by the protocol driver, asserted if the operation priority is high and deasserted if it is low. It can be
+  configured by setting the `pri-dir-gpios` property of the `nrf_radio_coex` device tree node. Currently only priority "low" is supported.
```

Comment:
> Currently only priority "low" is supported.

That's not true, the plugin is capable of signalling both priorities to the PTA: https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/mpsl/cx/thread/mpsl_cx_thread.c#L121-L123

Reply:
Some copy-paste error, fixed.